app/build.gradle

```diff
@@ -63,7 +63,7 @@ dependencies {
 
 
     testImplementation 'junit:junit:4.13.2'
-    testImplementation 'androidx.arch.core:core-testing:2.0.0'
+    testImplementation 'androidx.arch.core:core-testing:2.0.1'
```

Comment:
Apparently, it is 2.1.0 that is the latest following cirrus warning :sweat_smile: